includes/theme-navigation.php

```diff
@@ -134,13 +135,13 @@ public function bu_filter_anchor_attrs( $attrs ) {
 	 * @return array
 	 */
 	public function bu_navigation_filter_item_attrs( $item_classes ) {
-		if ( in_array( 'current_page_item', $item_classes, true ) ) {
```

Comment:
Oh, good call. I like the `array_diff()` return vs using `array_search()` each time. And I hadn't noticed the `page_item` class. Go for it! 👍